src/shared/bg_teamprogress.cpp

```diff
@@ -156,7 +147,7 @@ static void InformUnlockableStatusChanges( int *statusChanges, int count )
 			break;
 	}
 
-	CG_CenterPrint( text, 1.0f );
+	CG_CenterPrint( text.c_str(), 1.0f );
```

Comment:
Do we need a check for max str len or does CG_CenterPrint fail gracefuly? Maybe we should warn?

Otherwise LGTM

Reply:
I would expect the size to be bounded by the number of equipment anyway. I currently is readable on several lines without problem.

I don't think we'll reach some limits in text length in RmlUi anyway.

Reply:
Figure it can't hurt. We've been bitten by string length before and while it's probably not going to be a factor now, better to add a check than get bitten by it later...

Reply:
`CG_CenterPrint` does `Q_strncpyz( cg.centerPrint, str, sizeof( cg.centerPrint ) );` so this is here already